sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+| [`INSERT`](/sql-statements/sql-statement-insert.md) | Inserts new rows into a table. |
+| [`REPLACE`](/sql-statements/sql-statement-replace.md) | Replaces existing rows or inserts new rows. |
+| [`SELECT`](/sql-statements/sql-statement-select.md) | Retrieves rows from a table. |
+| [`TABLE`](/sql-statements/sql-statement-table.md) | Retrieves rows from a table. |
+| [`UPDATE`](/sql-statements/sql-statement-update.md) | Updates existing rows in a table. |
+| [`WITH`](/sql-statements/sql-statement-with.md) | Defines common table expressions. |
+
+## Transaction statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`BEGIN`](/sql-statements/sql-statement-begin.md) | Begins a new transaction. |
+| [`COMMIT`](/sql-statements/sql-statement-commit.md) | Commits the current transaction. |
+| [`ROLLBACK`](/sql-statements/sql-statement-rollback.md) | Rolls back the current transaction. |
+| [`SAVEPOINT`](/sql-statements/sql-statement-savepoint.md) | Sets a savepoint within a transaction. |
+| [`SET TRANSACTION`](/sql-statements/sql-statement-set-transaction.md) | Sets the characteristics of the current transaction. |
+| [`START TRANSACTION`](/sql-statements/sql-statement-start-transaction.md) | Starts a new transaction. |
+
+## Prepared statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`DEALLOCATE`](/sql-statements/sql-statement-deallocate.md) | Deallocates a previously prepared statement. |
+| [`EXECUTE`](/sql-statements/sql-statement-execute.md) | Executes a previously prepared statement. |
+| [`PREPARE`](/sql-statements/sql-statement-prepare.md) | Prepares a statement for execution. |
+
+## Administrative statements
+
+<CustomContent platform="tidb">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) | Cancels a DDL job. |
+| [`ADMIN CHECK [TABLE\|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) | Checks the integrity of a table or index. |
+| [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md) | Computes and checks the checksum of a table. |
+| [`ADMIN CLEANUP`](/sql-statements/sql-statement-admin-cleanup.md) | Cleans up residual files or resources. |
+| [`ADMIN PAUSE DDL`](/sql-statements/sql-statement-admin-pause-ddl.md) | Pauses DDL operations. |
+| [`ADMIN RESUME DDL`](/sql-statements/sql-statement-admin-resume-ddl.md) | Resumes DDL operations. |
+| [`ADMIN SHOW DDL [JOBS\|JOB QUERIES]`](/sql-statements/sql-statement-admin-show-ddl.md) | Shows DDL jobs or job queries. |
+| [`ADMIN SHOW TELEMETRY`](/sql-statements/sql-statement-admin-show-telemetry.md) | Shows telemetry data. |
+| [`ADMIN`](/sql-statements/sql-statement-admin.md) | Performs various administrative tasks. |
+| [`FLUSH TABLES`](/sql-statements/sql-statement-flush-tables.md) | Flushes a table. |
+| [`SET <variable>`](/sql-statements/sql-statement-set-variable.md) | Sets a variable. |
```

Comment:
```suggestion
| [`FLUSH TABLES`](/sql-statements/sql-statement-flush-tables.md) |  Included for MySQL compatibility. It has no effective usage in TiDB. |
| [`SET <variable>`](/sql-statements/sql-statement-set-variable.md) | Modifies one of TiDB's built-in variables |
```

Reply:
`SET` is not only for system variables, it is also for user variables.

```
mysql> SET @a := 5;
Query OK, 0 rows affected (0.00 sec)

mysql> SELECT @a * 3;
+--------+
| @a * 3 |
+--------+
|     15 |
+--------+
1 row in set (0.00 sec)
```

Reply:
Thanks for the verification. I've updated the comment.